peerchat/src/main/sqldelight/4.sqm

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE contact_state ADD COLUMN is_verified INTEGER NOT NULL DEFAULT 0;
```

Comment:
more descriptive name for the files 4..5..6..sqm?

Reply:
Yeah, should have done that. I continued with it since it already had 1, 2, 3 before I made a change to the table.

Reply:
Fine for now, if you can't fix it quickly.

Reply:
I’ll do it later!

Reply:
not possible to change the file names of database migration files